core/app/assets/js/ui/table/table.js

```diff
@@ -0,0 +1,70 @@
+import { useState, useEffect } from 'react';
+
+import { Context } from './table-context';
+import { arrayToClassName } from 'elementor-app/utils/utils.js';
+
+import TableHead from './table.head';
+import TableBody from './table-body';
+import TableRow from './table-row';
+import TableCell from './table-cell';
+import TableCheckbox from './table-checkbox';
+
+import './table.scss';
+
+export default function Table( props ) {
+	const getInitialSelections = () => {
+		const initialSelected = {};
+
+		props.initialSelected.forEach( ( value ) => initialSelected[ value ] = value );
+
+		return initialSelected;
+	},
+	[ selected, setSelected ] = useState( getInitialSelections() ),
```

Comment:
Check optimiation.

Reply:
Done (the 'selected' was changed from Object to an Array, therefore the function is no longer needed and was removed).